rasa/shared/importers/autoconfig.py

```diff
@@ -107,9 +107,12 @@ def _auto_configure(
     import pkg_resources
 
     if keys_to_configure:
+        collection_sentence = rasa.shared.utils.common.transform_collection_to_sentence(
```

Comment:
Can we rather do something like from ... import ...? Assigning the imports seems a bit intransparent to me